app/src/main/res/layout-land/story_chapter_view.xml

```diff
@@ -77,7 +77,7 @@
             android:layout_marginEnd="8dp"
             android:layout_marginBottom="4dp"
             android:fontFamily="sans-serif"
-            android:text="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? htmlContent : String.format(@string/chapter_prerequisite, viewModel.index, viewModel.previousChapterTitle)}"
+            android:text="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? htmlContent : @string/chapter_prerequisite_title_label(viewModel.index, viewModel.prerequisiteChapter.name)}"
```

Comment:
Per discussion in the team meeting, I believe we need to use HtmlParser in order to properly render the bolded text. Unfortunately, it's a bit obscure how to use it. It will actually significant complicate this PR, so what I suggest instead is:
1. Remove the bolded tags from the string for now (since we don't want them to show up if we can't render them)
2. @anandwana001 can you file an issue to track adding rich-text support for this view? It's a non-starter issue since it requires someone to add a custom view somewhere in this recycler view hierarchy & hook it up to the ViewComponent in order to support injecting HtmlParser and using it. Or, we can wait until #2013 is done, but we still need an issue to track making this new title string what it's supposed to be.

Reply:
Sounds good to me.

Reply:
Done. I removed the tags.

Reply:
https://github.com/oppia/oppia-android/issues/2252

Reply:
Thanks!